libcodechecker/libauth/cc_ldap.py

```diff
@@ -307,17 +341,36 @@ def auth_user(ldap_config, username=None, credentials=None):
         if group_base is None:
             LOG.error('Group base needs to be configured to'
                       'query ldap groups.')
-            return False
+            return []
 
-        member_pattern = ldap_config.get('groupMemberPattern')
-        if member_pattern is None or member_pattern == '':
-            member_pattern = '(member=$USERDN$)'
+        group_name_attr = ldap_config.get('groupNameAttr')
+        if group_name_attr is None:
+            LOG.error('groupNameAttr needs to be configured to'
+                      'query ldap groups.'
+                      'Its value must be the name'
+                      'attribute of the group.')
+            return []
 
-        member_pattern = member_pattern.replace('$USERDN$', user_dn)
-        member_query = '(& ' + group_pattern + member_pattern + ')'
+        # Attribute name must be ascii encoded
+        group_name_attr = group_name_attr.encode('ascii',
+                                                 'ignore')
+        attr_list = []
```

Comment:
This list and the `append()` call can be simplified, as you only append once to the list:

```py
attr_list = [group_name_attr]
```